### PR TITLE
Fix typo in "accumulating"

### PR DESCRIPTION
**Pull request by kinoute**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ task as a list of :class:`~celery_batches.SimpleRequest` instances.
 Some potential use-cases for batching of task calls include:
 
 * De-duplicating tasks.
-* Accumlating / only handling the latest task with similar arguments.
+* Accumalating / only handling the latest task with similar arguments.
```
Comment (clokep) [inline docs/index.rst:13]:
```suggestion
* Accumulating / only handling the latest task with similar arguments.
```

**Comment by clokep** [comment]
Thanks! The same typo seems to be in `README.rst` if you wouldn't mind fixing it there too?

**Comment by kinoute** [comment]
Done

File: README.rst [hunk 2]
```diff
@@ -18,7 +18,7 @@ timer and based on the number of queued tasks.
 Some potential use-cases for batching of task calls include:
 
 * De-duplicating tasks.
-* Accumlating / only handling the latest task with similar arguments.
+* Accumalating / only handling the latest task with similar arguments.
```
Comment (clokep) [inline README.rst:21]:
```suggestion
* Accumulating / only handling the latest task with similar arguments.
```

**Comment by clokep** [comment]
Meh, the build failures are unrelated to this but should be looked at. I'll merge this for now.

**Comment by clokep** [comment]
Thank you! 🎉

**Pull request merged**